resource_multi_week_work_time_from_contracts/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from . import resource_calendar
```

Comment:
`resource_mixin` should also be imported.

```suggestion
from . import resource_calendar
from . import resource_mixin
```